utils/build/virtual_machine/weblogs/python/provision_test-app-python-container-3.10.yml

```diff
@@ -0,0 +1,16 @@
+weblog:
+    name: test-app-python-container-3.10
+    exact_os_branches: [ubuntu24]
```

Comment:
I think we should try to run on multiple machines

Reply:
For now I'm following the guidance in the linked JIRA ticket, which is to only test these against latest ubuntu.